src/core/output.rs

```diff
@@ -1,40 +1,24 @@
-use std::{path::Path, process::Command};
+use flate2::{write::GzEncoder, Compression};
+use std::{fs::File, path::Path};
+use tar::Builder;
 
-use super::{CliError, LalResult};
+use super::{LalResult};
 
 /// Helper for stash and build
 pub fn tar(component_dir: &Path, tarball: &Path) -> LalResult<()> {
     info!("Taring OUTPUT");
-    let mut args: Vec<String> = vec![
-        "czf".into(),
-        tarball.to_str().unwrap().into(), // path created internally - always valid unicode
-        "--transform=s,^OUTPUT/,,".into(), // remove leading OUTPUT
-    ];
```

Comment:
Man, wish I had tests for this. Remember this was very subtle. I think we wanted compatibility with `tar` so the stuff we compressed could be piped to tar and extracted as well for debugging. Not sure if flate2 handled it correctly, and that might have been the reason for it (it should be in the history).

Reply:
> stuff we compressed could be piped to tar.

I'll add a test for this. After reshuffling, I think this makes sense to add this to the `export` tests.